ui/lib/config-ui/addon/components/messages/tab-page-header.hbs

```diff
@@ -35,7 +35,7 @@
           <LinkTo
             class={{unless @authenticated "active"}}
             @route="messages"
-            @query={{hash authenticated=false page=1}}
+            @query={{hash authenticated=false page=1 pageFilter="" status=null type=null}}
```

Comment:
When transitioning to different tabs, we want to make sure to reset the query params. I noticed there is a small bug similar to `FilterInput` where the component doesn't update until a full page refresh. The [workaround](https://github.com/hashicorp/vault/blob/267e06d94eeac74334bc796fb3becf8cb966272d/ui/lib/config-ui/addon/components/messages/page/list.js#L35-L53) for the `FilterInput` losing focus was to add a `routeDidChange` check in the list component, however, that's not possible for the SearchSelect. Open to suggestions!

https://github.com/hashicorp/vault/assets/30884335/817ffb29-3a7d-4722-b148-407f77b9a9a8

Reply:
I'm confused between the gif and the description. Is the problem that the filter isn't clearing, or that you want it to autofocus on one of the inputs?

Reply:
@chelshaw Sorry the description was confusing. The query params get reset correctly, however, the filter status stays preselected until there is a full page refresh and I think it's because the template doesn't get re-rendered. I logged the params in the `list` template and it logs `null` which is correct since we set the status query to null when transitioning to the other tab.